alma/alma-8.x/common/install_docker.sh

```diff
@@ -8,15 +8,8 @@ yum install -y moby-cli
 # Install NVIDIA Docker
 # Reference: https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html
```

Comment:
Please also update the installation reference if applicable

Reply:
This is the correct URL